tests/state_test_util.go

```diff
@@ -289,3 +330,117 @@ func rlpHash(x interface{}) (h common.Hash) {
 	hw.Sum(h[:0])
 	return h
 }
+
+// simulate gas price for Ethereum
+func simulateEthGasPrice(config *params.ChainConfig, json *stJSON) error {
```

Comment:
1. The function seems unpack the `gasPrice`. I was confused because the function name starts with `simluate`. What's the correct semantic you intended?
2. The `gasPrice` is set to `LowerBoundBaseFee`. Can you explain how this is effective?

Reply:
> The function seems unpack the gasPrice. I was confused because the function name starts with simluate. What's the correct semantic you intended?

Here sets the config so that tx senders pay as much fee as Ethereum. AFAIK, Kaia's gas price is base on only chain config not tx contains. Therefore gas price is calculated in Ethereum context, and the value is set to the config.

Reply:
https://github.com/ulbqb/kaia/blob/04126d29cbe605ae574d1272761bbab947adcafb/tests/state_test_util.go#L187
It's difficult to reproduce the gas price of Ethereum from the chain config so I'm going to directly set the gas price to txContext.